discord/commands/__init__.py

```diff
@@ -27,3 +27,4 @@
 from .core import *
 from .options import *
 from .permissions import *
+from .mixins import Invokable, _BaseCommand, BaseContext
```

Comment:
Related to my comment about using `__all__`, `from .mixins import *` should be used here as well